src/Jobs/Recruit.Vacancies.Jobs/AnalyticsSummaryProcessor/AnalyticsEventStore.cs

```diff
@@ -72,12 +73,15 @@ public class AnalyticsEventStore
 
         private readonly ILogger<AnalyticsEventStore> _logger;
         private readonly string _vacancyAnalyticEventsDbConnString;
+        private readonly AzureServiceTokenProvider _azureServiceTokenProvider;
+
         private RetryPolicy RetryPolicy { get; }
 
-        public AnalyticsEventStore(ILogger<AnalyticsEventStore> logger, string vacancyAnalyticEventsDbConnString)
+        public AnalyticsEventStore(ILogger<AnalyticsEventStore> logger, string vacancyAnalyticEventsDbConnString, AzureServiceTokenProvider azureServiceTokenProvider = null)
```

Comment:
Could we refactor this eventstore to inject the DbConnection like in other applications?